sapi/fpm/fpm/fpm_env.c

```diff
@@ -119,6 +119,9 @@ static char * nvmatch(char *s1, char *s2) /* {{{ */
 
 void fpm_env_setproctitle(char *title) /* {{{ */
 {
+#ifdef HAVE_SETPROCTITLE_FAST
```

Comment:
I might look better if it's `if defined` and `elif defined` so there are not so many endifs...